### ISSUE TITLE
different colors for `C_` in 4dm file in declaration

### ISSUE THREAD

**Issue by mesopelagique** [ayoubserti/lang-4d#6]
If see different color for  if there variable

```4d
C_TEXT
C_TEXT()
C_TEXT:C123()
C_TEXT:C123($a)
```
```4d
C_OBJECT()
C_OBJECT:C1216()
C_OBJECT:C1216("")
C_OBJECT:C1216($o)
C_OBJECT:C1216($o;$o1;$o2;$ob)
```
```4d
C_COLLECTION
C_COLLECTION($test;"")
C_COLLECTION:C123()
```
https://github.com/vdelachaux/classes/blob/49eafd112fc92f5d9748c8af7a23d4e2d2145384/Project/Sources/Methods/test_ob.4dm#L4
<img width="242" alt="Screenshot 2020-01-30 at 11 00 29" src="https://user-images.githubusercontent.com/59135882/73439344-bff1e580-434f-11ea-8816-ae83d9e04def.png">

```4d
C_TEXT:C284($t)
C_OBJECT:C1216($o;$o1;$o2;$ob)
C_COLLECTION:C1488($c)
```

**Comment by ayoubserti** [comment]
Thank you,

It's a bug in my regex

**Comment by mesopelagique** [comment]
Also C_VARIANT maybe never colorized

**Comment by ayoubserti** [comment]
@mesopelagique  Please reinstall extension and test. the fix was shipped in lang-4d-support@0.0.5
